src/components/studies/preview/Preview.tsx

```diff
@@ -118,6 +151,25 @@ const Preview: React.FunctionComponent<PreviewProps> = ({
       handleError(e!)
     }
   }
+
+  useEffect(() => {
+    const sessions = schedule.sessions
+    const allUniqueAssessments: Assessment[] = []
+    for (const session of sessions) {
```

Comment:
This works. But as a fun exercise - see if you can accomplish this using `reduce`